readthedocs/projects/urls/public.py

```diff
@@ -31,14 +31,6 @@
         public.project_downloads,
         name='project_downloads',
     ),
-    url(
-        (
-            r'^(?P<project_slug>{project_slug})/downloads/(?P<type_>[-\w]+)/'
-            r'(?P<version_slug>{version_slug})/$'.format(**pattern_opts)
-        ),
-        public.ProjectDownloadMedia.as_view(),
-        name='project_download_media',
-    ),
```

Comment:
This will break production, again. Please think about backwards compatibility when making large changes like this, we can't just randomly remove URL's in production.

Reply:
I just re-added. Actually, it breaks some tests as well because they started to return 404 :)

Reply:
I removed it originally because I thought it will interferes with the other url names `project_download_media` but they are in different URLCONF, so that does not happen.

Reply:
Hrm... Re-adding this here fixes some tests, but adds another problem. The URL of the `downloads` for the APIv3 is wrong. It returns `https://project.readthedocs.io/projects/project/downloads/pdf/v1.0/` when it should be `https://project.readthedocs.io/_/downloads/pdf/project/v1.0/`

Reply:
Oh, I think we should keep this URL but remove the `name=` from it.

So, if the user hits the URL directly, it will work. But if we need to generate the URL, we will use the new one :)